Makefile

```diff
@@ -11,8 +11,22 @@ PDF_GRAMMARLY = grammarly.pdf
 
 IMAGES := $(wildcard images/*.jpg images/*.pdf images/*.png)
 
+# spag is a phony rule, it generates index.html
+# from textidote
+.PHONY: spag
+
+
 all: ${PDF_PAPER}
 
+# spelling and grammar
+spag: paper.tex
+	# check that textidote exists.
+	@textidote --version
+	# allowed to fail since it throws error if we have grammar mistakes
+	-textidote --check en --output html paper.tex > index.html
+	python3 -m http.server
+
```

Comment:
Could you add `open http://localhost:8000/` to automatically open the webpage (instead of having to remember which port you need to connect to).

Reply:
Does `open`  exist across linux distros? I personally am aware of `xdg-open` which is what I use.